paws.common/R/client.R

```diff
@@ -98,11 +101,11 @@ resolver_endpoint <- function(service, region, endpoints, scheme = "https") {
 
 
 # client_config returns a ClientConfig configured for the service.
-client_config <- function(service_name, endpoints) {
-  s <- new_session()
+client_config <- function(service_name, endpoints, cfgs = NULL) {
+  s <- new_session(cfgs)
   region <- s$config$region
   if (s$config$endpoint != "") {
-    endpoint <- sprintf("https://%s", s$config$endpoint)
+    endpoint <-  s$config$endpoint
```

Comment:
This should add https if it isn't already in the endpoint string.

Reply:
The Go SDK and other SDKs have you specify the full endpoint with https:
`svc := ec2.New(sess, &aws.Config{Endpoint: aws.String("https://test.us-west-2.amazonaws.com")})`

Reply:
Ok, sounds good.